torch/testing/_internal/distributed/distributed_test.py

```diff
@@ -4112,6 +4116,18 @@ def _test_ddp_hook_parity(self, state, hook):
                     msg=f"Expected hook grad to be close to allreduce {avg[0, 0]}, but got {avg_hook[0, 0]}",
                 )
 
+        @sandcastle_skip_if(
```

Comment:
No need to add this test. It's already covered by `test_builtin_ddp_comm_hooks_nccl`.